app/src/pages/inside/productVersionPage/productVersionHeader/popoverTools/popoverTools.scss

```diff
@@ -0,0 +1,42 @@
+
+.popover-tools {
+  padding: 8px 0;
+  font-size: 13px;
+
+  &__item-button {
+    font-family: $FONT-ROBOTO-REGULAR;
+    font-weight: 400;
```

Comment:
In this repo, the `font-family` already includes the corresponding `font-weight`, so `font-weight` usage not needed here.

Reply:
removed both